tests/integration/src/test/java/org/apache/pulsar/tests/integration/presto/TestPulsarSQLBase.java

```diff
@@ -267,16 +268,33 @@ private void validateData(TopicName topicName, int messageNum, Schema schema) th
     }
 
     public ContainerExecResult execQuery(final String query) throws Exception {
+        return execQuery(query, null);
+    }
+
+    public ContainerExecResult execQuery(final String query, Map<String, String> extraCredentials) throws Exception {
         ContainerExecResult containerExecResult;
 
+        StringBuilder extraCredentialsString = new StringBuilder(" ");
```

Comment:
Init with capacity seems better.

Reply:
It seems it's difficult to determine the capacity of the string builder in advance. It seems not a big deal.